brave/src/test/java/brave/handler/MutableSpanTest.java

```diff
@@ -99,6 +99,18 @@
     );
   }
 
+  /** This is a compile test to show how the signature is intended to be used */
```

Comment:
This comment seems to be drifted on multiple tests so no need to fix now, but realize it doesn't seem correct (we have assertions it's not just a compile test).

Reply:
agreed!